src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -906,6 +897,109 @@ static bool usesReceiver(BoundExpression argument)
             }
         }
 
+        private RefKind GetRefKindForCapturedReceiverTemp(BoundExpression rewrittenReceiver)
```

Comment:
This method is extracted, but the case for `ref readonly` parameter was added (was blowing up in `StoreToTemp`). Added a non-extensions repro